packages/main/src/TableUtils.ts

```diff
@@ -99,6 +99,18 @@ const toggleAttribute = (element: HTMLElement, attribute: string, condition: boo
 	}
 };
 
+/**
+ * Checks if a given width is valid for a column.
+ *
+ * @param width Width string to check
+ * @returns {boolean} true if the width is valid, false otherwise
+ */
+const isValidColumnWidth = (width: string | undefined): width is string => {
```

Comment:
crazy return type

Reply:
Yeah, pretty neat functionality to ensure the type of the provided param

https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates